src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/Binding.Custom.Tests.csproj

```diff
@@ -23,6 +23,9 @@
     <None Include="project.json" />
   </ItemGroup>
   <ItemGroup>
+     <TestCommandLines  Condition="'$(TargetOS)'!='Windows_NT'" Include="sudo -E -n ./InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file /tmp/wcfrootca.crt"/>
```

Comment:
I'd prefer we find a way to avoid copy/pasting this same line into every project like this?  Can we put this somewhere like build.override.targets?  Or create a separate .props file included by any scenario test that needs this? Multiple copies should be avoided wherever possible.